engine/consensus/sealing/core_test.go

```diff
@@ -722,3 +732,68 @@ func (s *ApprovalProcessingCoreTestSuite) TestRepopulateAssignmentCollectorTree(
 		require.Equal(s.T(), approvals.VerifyingApprovals, collector.Collector.ProcessingStatus())
 	}
 }
+
+// TestRepopulateAssignmentCollectorTree_RootSealingSegment tests that the sealing
+// engine will be initialized correctly when bootstrapping with a root sealing
+// segment with multiple blocks, as is the case when joining the network at an epoch
+// boundary.
+//
+// In particular, the assignment collector tree population step should ignore
+// unknown block references below the root height.
+//
```

Comment:
```suggestion
```

Reply:
This is intentional -- whitespace in longer comments for easier reading ;)